pages/_app.tsx

```diff
@@ -11,8 +11,8 @@ import { firebaseClient } from "../utils/firebaseClient";
 require("../styles/globals.css");
 require("@solana/wallet-adapter-react-ui/styles.css");
 
-const auth = getAuth(firebaseClient);
-async function callback(data: Record<string, string>) {
+export const auth = getAuth(firebaseClient);
```

Comment:
We should move this "auth" elsewhere. Maybe to the firebaseClient file directly?

exporting this from _app.tsx feels ugly, this page shouldn't need to hold the responsibility for managing the fb auth istance